compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -651,6 +651,12 @@ class Definitions {
     @tu lazy val Enum_ordinal: Symbol = EnumClass.requiredMethod(nme.ordinal)
 
   @tu lazy val EnumValuesClass: ClassSymbol = requiredClass("scala.runtime.EnumValues")
+  @tu lazy val EnumValueClass: ClassSymbol = requiredClass("scala.runtime.EnumValue")
+
+  @tu lazy val EnumValueSerializationProxyClass: ClassSymbol = requiredClass("scala.runtime.EnumValueSerializationProxy")
+    @tu lazy val EnumValueSerializationProxyConstructor: TermSymbol =
+      EnumValueSerializationProxyClass.requiredMethod(nme.CONSTRUCTOR, List(ClassType(TypeBounds.empty)))
```

Comment:
The constructor takes two arguments: a Class[?] and an Int, so that list of argument looks wrong to me;

Reply:
good spot thank you :)